src/js/personalization/va-profile-beta/utils/address.js

```diff
@@ -70,7 +70,7 @@ function getInferredAddressType(address) {
  * @param {Address} address
  * @returns {Address}
  */
-function expandAddress(address) {
+export function expandAddress(address) {
```

Comment:
I'm a little unclear on how country is handled for military addresses, it is deleted here, and defaulted to US above. Perhaps some comments about this would be helpful, if possible.